modules/core/src/main/java/org/apache/ignite/internal/jdbc/thin/JdbcThinTcpIo.java

```diff
@@ -126,6 +126,9 @@ public class JdbcThinTcpIo {
     /** Random. */
     private static final AtomicLong IDX_GEN = new AtomicLong(new Random(U.currentTimeMillis()).nextLong());
 
+    /** Default nested tx mode for compatibility. */
+    private static final String DEFAULT_NESTED_TX_MODE = "ERROR";
```

Comment:
Can we use single constant here and in `JdbcThinTcpio`?

Reply:
Good point!